dev/breeze/src/airflow_breeze/commands/ci_image_commands.py

```diff
@@ -443,27 +448,18 @@ def build_ci_image(verbose: bool, dry_run: bool, ci_image_params: BuildCiParams)
     if not ci_image_params.force_build and not ci_image_params.upgrade_to_newer_dependencies:
         if not should_we_run_the_build(build_ci_params=ci_image_params, verbose=verbose):
             return 0, f"Image build: {ci_image_params.python}"
-    run_command(
```

Comment:
We are not using "inline" cache any more, so we do not have to remove the image when building. This is great because it resulted in disappearing image if you broke the build with ctrl-c